content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,110 @@
+
+
+## Upgrading to Gateway
+
+1. [Install a Gateway controller](https://gateway-api.sigs.k8s.io/guides/#installing-a-gateway-controller) OR [install the Gateway API CRDs manually](https://gateway-api.sigs.k8s.io/guides/#installing-gateway-api) 
+
+2. install [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway)
+
+```
+go install github.com/kubernetes-sigs/ingress2gateway@latest
+```
+Provided you have a Go development environment installed, that command installs `ingress2gateway` to `$(go env GOPATH)/bin/ingress2gateway`.
+
+Alternatively, you can build it from source by following [Build from source](https://github.com/kubernetes-sigs/ingress2gateway/tree/95b4a0316bac8d28d72b871a210bbc9bd6191eb7#build-from-source)
+
+3. Once the tool is installed, you can use it to convert the ingress resources in your cluster to GatewayAPI resources.
+
+```
+ingress2gateway print
+```
+
+This above command will:
+
+1. Read your Kube config file to extract the cluster credentials and the current active namespace.
+1. Search for ingresses and provider-specific resources in that namespace.
+1. Convert them to Gateway-API resources (Currently only Gateways and HTTPRoutes).
+For other options refer run the tool with `-h` or refer to https://github.com/kubernetes-sigs/ingress2gateway#options
+
+4. Review the converted Gateway API resources, validate them, and then apply them to your cluster.
+
+5. Update your DNS to point to the new gateway. 
+
+Note: make sure you test the newly created gateway before shifting the shifting 100% of the traffic.
+
+6. Make sure no traffic goes through your ingress and delete it.
+
+## Wrapping up
+Achieving reliable, scalable and extensible networking has always been a challenging objective.
+The Gateway API is designed to improve the current Kubernetes networking standards like ingress and/or implementation specific annotations and CRDs.
+
+It is a Kubernetes standard API, consistent across different platforms and implementations and most importantly it is future proof. The Gateway API is considered the Ingress’ successor, it has a sub team under SIG Network that actively work on it and manage the ecosystem and it is likely to receive more updates and community support.
+
+Ingress2gateway tool helps to ease the migration process for you, converting an existing Ingress object to equivalent objects from Gateway. To make migrations even easier, Ingress2gateway has recently added extensions to enable converting from provider-specific solutions, enabling it to translate implementation-specific CRDs or Ingress annotations to the closest Gateway API equivalent.
```

Comment:
I think you've already made a good case for why people should upgrade to Gateway API to start this post, I'd recommend focusing on what is coming up next - any future plans for Ingress2Gateway, the upcoming v1.0 release of Gateway API, etc.

Reply:
made the changes, check "The Road Ahead" section